stable/admin/templates/statefulset.yaml

```diff
@@ -112,6 +112,10 @@ spec:
       {{- end }}
         args:
           - "nuoadmin"
+          {{- if .Values.admin.evicted.servers}}
```

Comment:
If you decide to keep the variable nesting, then we might want to also check that `.Values.admin.evicted` is not `nil`.

Reply:
I can see a top level evicted _might_ be useful when we come to looking at failover automation, but it's too early to say given there is no solution yet.

Reply:
well spotted. Fixed